### DISCUSSION TITLE
How can I add a locale that is not supported yet?

### DISCUSSION THREAD

**Discussion by MeinhardD** [moment/luxon#1150] [General]
Hello,

Is it possible to add a locale that isn't supported yet here? Or do the locales come from somewhere else? As in, do the names of months and days come from somewhere else, or is there a file or files in luxon, where I can add the locale of my native language?

**Comment by icambron** [comment]
The locales come from an API in the javascript runtime: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat. You'd have to get your native language added to the Intl Data the browser (and Nodejs) provide
